Wrappers/Python/ccpi/astra/operators/AstraProjectorSimple.py

```diff
@@ -63,7 +76,30 @@ def domain_geometry(self):
     def range_geometry(self):
         return self.sinogram_geometry    
     
+    def FBP(self, DATA, filter_type):
+    
+        self.fbp.filter_type = filter_type
+        self.fbp.set_input(DATA)
+        out = self.fbp.get_output()
+        
+        return out    
+    
```

Comment:
Should this stay or be deleted to have only FBP as a processor and not as a method in the operator? Can't remember what we decided.